modules/administration-guide/partials/con_devworkspace-operator.adoc

```diff
@@ -0,0 +1,34 @@
+[id="devworkspace-operator_{context}"]
+= {devworkspace} operator
+
+include::example$snip_using-devworkspace-warning.adoc[]
+
+The {devworkspace} operator extends {orch-name} to provide {devworkspace} support.
+
+{devworkspace} Operator:: 
+
+Manages the lifecycle of the {devworkspace} {orch-name} resources.
+
+{devworkspace} Custom Resource Definition:: 
+
+Defines {orch-name} objects: {devworkspace} Custom Resources.
+
+{devworkspace} Custom Resources:: 
+
+{orch-name} objects defining the User workspaces.
+
+{devworkspace} controller:: 
+
+Manages {devworkspace} Custom Resources and controls the running state of the User workspaces.
```

Comment:
```suggestion
{The {devworkspace} operator extends {orch-name} to provide {devworkspace} support. It's components are:

{devworkspace} Custom Resource Definition:: 

Defines the {devworkspace} {orch-name} object. The definition is automatically generated from the Devfile specification.

{devworkspace} controller:: 

Creates and controls the necessary {orch-name} objects to run a {devworkspace}. Including `Pods`, `Services`, and `PeristentVolumes`.
It is a Go application.

On a cluster with the {devworkspace} operator, it is possible to create {devworkspace} Custom Resources (CR). A {devworkspace} CR is a {orch-name} representation of a Devfile. It defines a User workspaces in a {orch-name} cluster.
```

A Kubernetes operator is composed by a controller and some CRDs. If the operator is deployed on a cluster, the Kubernetes API is extended to manage objects defined by the CRD. I have updated the description to reflect that.